Frontend/ui-common/src/Components/SelectListModal/View.purs

```diff
@@ -68,74 +68,103 @@ view push config =
           , background Color.white900
           , padding (Padding 20 20 20 0)
           ][  headingText config push ,
-              cancelationReasonsList push config
+              optionListView push config
            ]
          ]
 
 
 headingText :: forall w . Config -> (Action -> Effect Unit) ->  PrestoDOM (Effect Unit) w
-headingText config push =
- linearLayout
- [ width MATCH_PARENT
- , height WRAP_CONTENT
- , orientation VERTICAL
- , clickable true
- , onClick ( \action -> do
-            _ <- pure $ hideKeyboardOnNavigation true
-            pure unit
-          ) (const NoAction)
- , disableClickFeedback true
- ][ textView
-    [ width MATCH_PARENT
+headingText config push = 
+  linearLayout
+  [
+    width MATCH_PARENT
+  , height WRAP_CONTENT
+  , orientation HORIZONTAL
+  ][
+    linearLayout
+    [
+      height WRAP_CONTENT
+    , width WRAP_CONTENT
     , height WRAP_CONTENT
-    , text config.headingText
-    , color Color.black800
     , orientation HORIZONTAL
-    , gravity CENTER
-    , textSize FontSize.a_22
-    , fontStyle $ FontStyle.bold LanguageStyle
-    ],
-    textView
+    , gravity LEFT
+    , margin $ MarginTop 4
+    , visibility if config.chevronVisibility then VISIBLE else GONE
+    , onClick push (const OnGoBack)
+    , padding (PaddingRight 12)
+    ][
+      imageView
+      [ height $ V 22
+      , width $ V 22
+      , imageWithFallback "ny_ic_chevron_left,https://assets.juspay.in/nammayatri/images/common/ny_ic_chevron_left.png"
+      , margin $ MarginTop 4
+      , color Color.black900
+      , fontStyle $ FontStyle.semiBold LanguageStyle
+      ] 
+    ]   
+    ,linearLayout
     [ width MATCH_PARENT
     , height WRAP_CONTENT
-    , text config.subHeadingText
-    , orientation HORIZONTAL
-    , padding (PaddingTop 4)
-    , gravity CENTER
-    , textSize FontSize.a_14
-    , color Color.black700
-    , fontStyle $ FontStyle.regular LanguageStyle
-    ],
-    linearLayout
-    [ height WRAP_CONTENT
-    , width WRAP_CONTENT
-    , padding (Padding 4 12 4 12)
-    , margin (Margin 2 16 0 0)
     , orientation VERTICAL
-    , onClick ((\action -> do
-        _ <- push action
-        _ <- setText' (getNewIDWithTag "OtherReasonEditText") ""
-        _ <- setText' (getNewIDWithTag "TechGlitchEditText") ""
-        pure unit
-    )) ( const ClearOptions)
-    , visibility case config.activeReasonCode of
-                    Just reasonCode -> if ( reasonCode == "OTHER" || reasonCode == "TECHNICAL_GLITCH") then VISIBLE else GONE
-                    _               -> GONE
-    ][  textView
-        [ width WRAP_CONTENT
+    , weight 1.0
```

Comment:
@Akash2172 height , width and weight all three are mentioned here so remove weight if not required or else remove the required attribute as weight won't do anything here.